test/langtools/tools/javac/diags/examples/NoTitle.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2012, 2022, Oracle and/or its affiliates. All rights reserved.
```

Comment:
2012? Is it an old file from the previous incarnation of the `@spec` tag?

Reply:
From the file it was derived from.   It's always a toss-up when cloning and modifying an old file.  2012 removed.